### ISSUE TITLE
Chapter on Facilitating Collaborations at Remote Events

### ISSUE THREAD

**Issue by sgibson91** [the-turing-way/the-turing-way#1005]
## Detailed description
 * This issue is a place to discuss matters relating to the writing of the chapter on Facilitating Collaboration at Remote Events.
 * This chapter is being written in _pull_request_number_ **TBD**
 * This chapter is being developed on the `1005/chapter/remote-events` branch.

### Table of contents

Open to discussion/rearranging!

1. Activity types / Use cases (Mix and match these to build your online event!)
   1. workshop
   2. hack event
   3. poster session
   4. prototyping/ideas session
   5. presentations (including lightning talks)
   6. pre-recorded presentations
   7. panel sessions
   8. async collaboration on a text
   9. Tools for pair-programming
      1.  Requirements
          1.  breakout rooms
          2.  chat
          3.  video/audio (where can these be tested?)
          4.  VCS integration
          5.  live streaming
          6.  whiteboard
          7.  Recording tools for pre-recorded talks
              1.  Available tools (link up with https://github.com/alan-turing-institute/the-turing-way/issues/939)
                  1.  feature matrix
                  2.  account needed? How can people join?
                  3.  installation needed?
                  4.  paid-for/free
                  5.  open source
      2.  How to replicate/simulate the social aspects
          1.  Zoom backgrounds ;)
          2.  Virtual water cooler/coffee break
          3.  Virtual pub quizzes
      3.  Things you wouldn’t do in person but can do virtually
          1.  collaborative notetaking
          2.  pre-recording talks
      4.  Scheduling (may be an issue particular to online events because people will be in their own timezones, not all in one)

### Resources

* Feature matrix of tooling: https://docs.google.com/spreadsheets/d/1g3huaRREpZA_HM87lO8DimerkDZvH1-R3G3e3MF0orQ/edit#gid=0
* Idea 5 outline document: https://docs.google.com/document/d/1o-wQtRObd9iu-7jqz3KDV0RqCGVzvXIVpLxvddeOXm4/edit
* Master Hackday pitch document: https://docs.google.com/document/d/1HY0ByqKZmAi23bt0BwTbAzX1ym8X_hd6hvpa8VRa9OY/edit
* CW20 hackday hackmd: https://hackmd.io/@sgibson91/turing-way-templating

### Discussion points/related issues

Due to the current COVID-19 crisis, remote work has become the default mode of working for most people in research, but it has also been common practice for different people and projects before that. How can we, working remotely, maintain the communities in this mode, and make sure that the community and the people involved in it remain healthy, and productive, and have all the information and technical support they need? In open source, this sort of remote work has been common. How can we disseminate the lessons learned in these communities to research groups and teams who are experiencing remote work, collaboration, teaching for the first time?

## Current status
This chapter is empty. If anyone would like to make a start they are more than welcome to do so.

**2020/04/02:** Currently being hacked on at CW20 hackday!! 😄 

 - [ ] Write chapter outline
 - [ ] Add material to the chapter
 - [ ] Combine materials into a readable chapter
 - [ ] Proofread
 - [ ] Request reviews
 - [ ] Address reviews
 - [ ] Merge to master branch.

## Updates
To avoid that others have to read through the full thread of comments, update the initial issue with important updates (e.g. decisions taken) regularly so that all the important information is available at one glance.

### CW20 team

- @sgibson91 
- @sdruskat
- @fmcooper
- @JoannaLeng

**Comment by sgibson91** [comment]
### Activity Type template

```
### <Activity Name>

By <activity> we mean ...

Usually, at in-person <activities> you have ...

But for online <activities> you need to think about how to do this or that ...

*Requirements*

*Solutions* Description of solutions 

*Available tools*

- *Feature matrix / Open Source? / Pricing*
```

**Comment by malvikasharan** [comment]
@sgibson91 Can we please connect this issue with #960 which is already a chapter?

**Comment by sgibson91** [comment]
Yes!

**Comment by malvikasharan** [comment]
@sgibson91 Please see if this chapter should be a chapter on its own or be added to the chapter on remote collaboration, currently in the PR #1566

**Comment by sgibson91** [comment]
Hmmm, tough one. It's more about virtual events, so I think it's its own chapter under collaboration? What do you think?

**Comment by sdruskat** [comment]
> Hmmm, tough one. It's more about virtual events, so I think it's its own chapter under collaboration? What do you think?

FWIW, I think this makes sense :).

**Comment by malvikasharan** [comment]
Makes sense. The other PR discusses how to facilitate/organise/manage/participate/chair virtual and remote events etc.

**Comment by sgibson91** [comment]
Also, merging the 2 would just make the chapter _huge_ 😅 Maybe this chapter precedes the other? Like "these are the types of events you could host/include, and now here's how to organise and chair them"?

**Comment by malvikasharan** [comment]
Sounds good to me, we can see later if we need to move some subhcapters from other chapter to this.

**Comment by JoannaLeng** [comment]
Lets not have a huge chapter :-)
